### PR TITLE
[Share] Enter password: toggle visibility of password #1171

### PR DESCRIPTION
**Pull request by vnidens**
Pull request for #1171

### CODE REVIEW

**Comment by tobiasKaminsky** [comment]
Thank you for your contribution! It is always nice to have new contributors. 

It is working fine, but I would assume that the icon will switch from "open eye" to "closed eye / crossed eye"?

**Comment by vnidens** [comment]
I've just supposed that it should look the same as in the login activity (as you've mentioned in the issue comments). At this moment existing login activity has the same visibility indicator as your screenshots.

If you don't mind I would update both views with the "open eye" <-> "crossed eye" indicator.

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky unfortunately no.... Google changed that and also on the login screen you'll see that now Android "only" changes the opacity of the eye icon. So that is a platform matter :/

**Comment by AndyScherzinger** [comment]
@vnidens sorry for the confusion, writing my post took too long...
@vnidens @tobiasKaminsky I am fine either way!

**Comment by tobiasKaminsky** [comment]
https://material.io/guidelines/components/text-fields.html#text-fields-input-types
Search for "password" there it is still with "crossed eye".

But for me either ways are fine.

**Comment by vnidens** [comment]
Updated to use crossed eye indicator for both password inputs

**Comment by tobiasKaminsky** [comment]
Great work, but I think the icons are swapped?
At least, on KDE wifi manager it is the other way around:

you can read the password, click on "hide" to cover it....
screenshot_20170718_153454 screenshot_20170718_153514

File: src/main/res/layout-land/account_setup.xml [hunk 1]
```diff
@@ -228,7 +229,8 @@
 					android:id="@+id/input_layout_account_password"
 					android:layout_width="match_parent"
 					android:layout_height="wrap_content"
-					android:visibility="gone">
+					android:visibility="gone"
+					app:passwordToggleDrawable="@drawable/password_visibility_selector">
```
Comment (tobiasKaminsky) [inline src/main/res/layout-land/account_setup.xml]: remove white space after =

File: src/main/res/layout-land/account_setup.xml [hunk 2]
```diff
@@ -19,6 +19,7 @@
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+	xmlns:app="http://schemas.android.com/apk/res-auto"
```
Comment (tobiasKaminsky) [inline src/main/res/layout-land/account_setup.xml]: not needed whitespace

**Comment by AndyScherzinger** [comment]
Nope, as you refered to https://material.io/guidelines/components/text-fields.html#text-fields-input-types it is right. It doesn't reflect the state but the action that will happen when you click.

Go home KDE, your drunk!

**Comment by AndyScherzinger** [comment]
Not sure what happens since I couldn't test it yet, so asking: What will happen on Android 4.0? Since the ic_show is available as png also while the ic_hide icon is vector only and I am not sure if this will break 4.0 since I think proper vector support came in post 4.0, right?

**Comment by tobiasKaminsky** [comment]
> Nope, as you referred to https://material.io/guidelines/components/text-fields.html#text-fields-input-types it is right.

Right, let's stick to material design :+1:

**Comment by vnidens** [comment]
@AndyScherzinger The gradle plugin converts xml vector drawables to pngs if the minSdkVersion < 21 so it's fine.
I can't check with 4.0.3 (API 14) since the emulator is veeery slow on my machine but here is a screenshot from 4.1 (API 16) which is also < 21.
screenshot_1500386894

**Comment by tobiasKaminsky** [comment]
I also tested it successfully with 4.0.4 but emulator of 4.0 does not work.

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky 4.0.4 is perfectly fine! As we don't support 4.0 but only 4.0.x and there it is basically 4.0.3. So 👍

**Comment by AndyScherzinger** [comment]
@vnidens right, totally forgort about the build taking care of that... Thanks for the hint. I'll test drive it later today

**Comment by tobiasKaminsky** [comment]
> @tobiasKaminsky 4.0.4 is perfectly fine! As we don't support 4.0 but only 4.0.x and there it is basically 4.0.3.

We have minSdk set to 14 in AndroidManifest, which is Android 4.0, 4.0.1, 4.0.2, acoording to
https://developer.android.com/guide/topics/manifest/uses-sdk-element.html

**Comment by tobiasKaminsky** [comment]
> @tobiasKaminsky do you have a pre-12 instance so I can test the login?

I have them only local :-/

**Comment by AndyScherzinger** [comment]
👍 fine code wise, also tested the link password part, codacy and lint are happy too ❤️ 

@tobiasKaminsky rebased to latest master so need another thumbs up :)

**Comment by tobiasKaminsky** [comment]
:+1: 
Thank you very much for this contribution, @vnidens! :tada:

**Pull request merged**